.scripts/run_docker_build.sh

```diff
@@ -59,7 +59,7 @@ if [ -z "${CI}" ]; then
 fi
 
 export UPLOAD_PACKAGES="${UPLOAD_PACKAGES:-True}"
-docker run ${DOCKER_RUN_ARGS} \
+nvidia-docker run ${DOCKER_RUN_ARGS} \
```

Comment:
This is a machine-specific change that makes the GPUs recognized by docker. `nvidia-docker` is discontinued as NVIDIA drivers are now natively supported by docker, but on our machine somehow it has some problems which I haven't diagnosed, and I noticed the old `nvidia-docker` still works.

Reply:
As docker is heavily used on this machine, I prefer not to fix/update docker for now.